mmocr/datasets/kie_dataset.py

```diff
@@ -214,13 +214,20 @@ def pad_text_indices(self, text_inds):
 
     def compute_relation(self, boxes):
         """Compute relation between every two boxes."""
-        x1, y1 = boxes[:, 0:1], boxes[:, 1:2]
-        x2, y2 = boxes[:, 4:5], boxes[:, 5:6]
+        # Get minimal axis-aligned bounding boxes for each of the boxes
+        # yapf: disable
+        bboxes = np.concatenate(
+            [boxes[:, 0::2].min(axis=1, keepdims=True),
+             boxes[:, 1::2].min(axis=1, keepdims=True),
+             boxes[:, 0::2].max(axis=1, keepdims=True),
+             boxes[:, 1::2].max(axis=1, keepdims=True)],
+            axis=1).astype(np.float32)
+        x1, y1 = bboxes[:, 0:1], bboxes[:, 1:2]
```

Comment:
```suggestion
        # yapf: enable
        x1, y1 = bboxes[:, 0:1], bboxes[:, 1:2]
```
Otherwise yapf would ignore the rest of the code as well